### PR TITLE
qemu.tests.single_driver_install: Update virtio serial device name for different hosts.

### PR DESCRIPTION
**Pull request by CongLi**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1379045

### CODE REVIEW

**Comment by CongLi** [comment]
17:56:13 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.single_driver_install.with_vioserial

File: qemu/tests/cfg/single_driver_install.cfg [hunk 1]
```diff
@@ -56,7 +56,9 @@
             remove_image_stg = yes
         - with_vioserial:
             driver_name = vioser
-            device_name = "VirtIO-Serial Driver"
+            device_name = "VirtIO Serial Driver"
+            Host_RHEL.m6, Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2:
+                device_name = "VirtIO-Serial Driver"
```
Comment (suqinhuang) [inline qemu/tests/cfg/single_driver_install.cfg]: only rhel7.3 use "VirtIO Serial Driver" right?

**Comment by ngu-niny** [comment]
Test pass in both RHEL7.2 and RHEL7.3 hosts with the correct virtio-win version respectively:
PASS 1-smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.Win2012.x86_64.io-github-autotest-qemu.single_driver_install.with_vioserial
PASS 1-smp_8.16384m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.single_driver_install.with_vioserial

So ack the patch:
Acked by: Nini Gu <ngu@redhat.com>

**Comment by CongLi** [comment]
@suqinhuang 

More comments on it?

**Comment by suqinhuang** [comment]
LGTM

**Comment by CongLi** [comment]
Thanks @ngu-niny @suqinhuang .

**Pull request merged**